config/express.js

```diff
@@ -11,14 +11,29 @@ const flash = require("connect-flash");
 const helpers = require("view-helpers");
 const bodyParser  = require('body-parser');
 const methodOverride = require('method-override');
-const morgan = require('morgan');
 const cookieParser = require('cookie-parser');
 const Raven = require('raven');
 const moment = require('moment');
+const morgan = require('morgan');
 
 module.exports = (app, config, passport) => {
   app.set("showStackError", true);
   app.locals.moment = moment;
+
+  // use morgan for logging
+  app.use(morgan('dev', {
+    skip: function (req, res) {
+      return res.statusCode < 400;
+    }, stream: process.stderr
+  }));
+
+
+  // use morgan for logging
+  app.use(morgan('dev', {
+    skip: function (req, res) {
+      return res.statusCode >= 400
+    }, stream: process.stdout
+  }));
```

Comment:
Can these 2 `app.use` calls be combined somehow?

Reply:
Yes, we definitely could